ipfabric/models.py

```diff
@@ -12,7 +13,95 @@
 IGNORE_COLUMNS = {"id"}
 
 
+class Site(BaseModel):
```

Comment:
This was all moved to `snapshot_models.py`  I don't think you merged correctly?

Reply:
fix in latest commit [7ff8abc](https://github.com/community-fabric/python-ipfabric/pull/214/commits/7ff8abca01f2572e0b002661a37ecc89f841731f)